apps/src/lib/levelbuilder/lesson-editor/AddResourceDialog.jsx

```diff
@@ -232,8 +252,9 @@ export default class AddResourceDialog extends Component {
                 <input
                   type="checkbox"
                   style={styles.checkboxInput}
-                  name="pdf"
-                  value={this.state.pdf}
+                  name="includeInPdf"
+                  value={this.state.includeInPdf}
+                  checked={this.state.includeInPdf}
```

Comment:
I think you can omit `value`, here and above, when `checked` is present. this is just based on looking at other code examples so I may be missing something though.

Reply:
Because this is still a form submission, I'm having issues with the request being correct even though the state is correct. Now that the lesson edit page uses ajax, I'm going to file a ticket to move this to ajax as well. I thought of doing that as part of this PR but I thought it would get a bit too big

Reply:
https://codedotorg.atlassian.net/browse/PLAT-525